plugins/restart.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: "restart"
+spec:
+  platforms:
+  - head: https://github.com/achanda/kubectl-restart/archive/v0.0.1.zip
+    files:
+    - from: "/kubectl-restart-0.0.1/*"
```

Comment:
You can probably do `/kubectl-restart-*/*` here so the newer versions can easily be maintained without updating this. But don't forget to test this with `kubectl plugin install --source=[...]`